ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +133,77 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        title="Instructions to add a new Resource in Hub"
+        isOpen={isResInstModallOpen}
+        onClose={() => setResInstModalOpen(false)}
+        width={'50%'}
+      >
+        <Grid>
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Add a new Resource through Catalogs and follow the below steps
+            </Text>
+            <Text>
+              Add a new resource to Hub by creating a pull request to available Catalogs with the
+              new resource which must be following the guidelines
+              <Text
+                component={TextVariants.a}
+                target="_blank"
+                href="https://github.com/tektoncd/catalog#catalog-structure"
+              >
+                {' '}
+                here
+              </Text>
+            </Text>
+            <Text>Available catalogs are:</Text>
```

Comment:
Should we show the list of catalog available, because we are already showing it them in filter

Reply:
> Should we show the list of catalog available, because we are already showing it them in filter

cc: @vinamra28 @piyush-garg

Reply:
yeah we should be keeping that